manifests/08-clusteroperator.yaml

```diff
@@ -3,3 +3,9 @@ kind: ClusterOperator
 metadata:
   name: console
 spec: {}
+status:
+  versions:
+    - name: operator
```

Comment:
What does `snapshot` and `snapshot_openshift` mean?
Should it be `snapshot-console` and` snapshot-console-operator`?  Guessing no,just curious the significance.

Looking at [kube-apiserver-operator](https://github.com/openshift/cluster-kube-apiserver-operator/blob/master/manifests/0000_10_kube-apiserver-operator_07_clusteroperator.yaml) quick